source/app.d

```diff
@@ -89,7 +94,11 @@ void githubHook(HTTPServerRequest req, HTTPServerResponse res)
         // no need to trigger the checker for failure/pending
         if (state == "success")
         {
-            searchForAutoMergePrs(repoSlug);
+            if (Clock.currTime - lastFullPRCheck >= timeBetweenFullPRChecks)
+            {
+                searchForAutoMergePrs(repoSlug);
```

Comment:
Simple throttling for querying over all open PRs. Currently it's throttle to `15.minutes`